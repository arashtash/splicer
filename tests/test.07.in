Integer ultricies risus et viverra fermentum. Nam et metus EOS
non arcu imperdiet placerat. Pellentesque ullamcorper urna vel magna pharetra, EOS
vitae libero. Etiam nibh tortor, posuere ac tincidunt EOS
sem eros, nec varius purus dapibus eu. Vestibulum eget nunc EOS
felis in urna. In viverra sem eros, nec varius EOS
eu posuere pellentesque, urna nisi accumsan ante, ut facilisis EOS
ultrices at, interdum vitae libero. Etiam nibh tortor, posuere EOS
commodo at nibh. Integer a fringilla lacus, tristique laoreet ligula. EOS
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque sed EOS
nec, malesuada vel arcu. Fusce auctor mi et placerat EOS
torquent per conubia nostra, per inceptos himenaeos. EOS
auctor mi et placerat dignissim. Fusce sit amet EOS
a fringilla lacus, tristique laoreet ligula. Pellentesque a sagittis EOS
blandit, sodales est. Duis tincidunt, turpis eu posuere pellentesque, urna EOS
est, suscipit elementum ante porttitor iaculis. Nulla facilisi. EOS
dictum feugiat id at ligula. Class aptent taciti EOS
adipiscing elit. Pellentesque sed dolor non arcu imperdiet placerat. EOS
et metus dignissim, gravida lacus blandit, sodales est. Duis tincidunt, EOS
nec neque sed neque viverra congue id quis EOS
ante porttitor iaculis. Nulla facilisi. Nam nec neque sed neque EOS
id at ligula. Class aptent taciti sociosqu ad litora EOS
molestie pharetra arcu, a rutrum libero tincidunt sed. Aliquam felis EOS
tellus mauris. Duis a lectus ligula. Nam viverra quam est, EOS
Nam viverra quam est, suscipit elementum ante porttitor iaculis. EOS
ex, cursus non ultrices at, interdum vitae libero. EOS
purus dapibus eu. Vestibulum eget nunc ac ligula dictum feugiat EOS
viverra congue id quis felis. Integer ultricies risus et EOS
laoreet ligula. Pellentesque a sagittis neque. Curabitur molestie pharetra EOS
Class aptent taciti sociosqu ad litora torquent per conubia nostra, EOS
at varius nec, malesuada vel arcu. Fusce auctor EOS
neque. Curabitur molestie pharetra arcu, a rutrum libero EOS
ut facilisis turpis felis in urna. In viverra EOS
urna nisi accumsan ante, ut facilisis turpis felis EOS
placerat dignissim. Fusce sit amet tellus mauris. Duis a lectus EOS
libero tincidunt sed. Aliquam felis sapien, fringilla at EOS
felis sapien, fringilla at varius nec, malesuada vel arcu. EOS
et viverra fermentum. Nam et metus dignissim, gravida EOS
nibh tortor, posuere ac tincidunt eget, commodo at nibh. Integer EOS
placerat libero ultrices. Nunc ante ex, cursus non ultrices at, EOS
nunc ac ligula dictum feugiat id at ligula. Class EOS
urna vel magna pharetra, et placerat libero ultrices. Nunc ante EOS
lectus ligula. Nam viverra quam est, suscipit elementum EOS
EOS
