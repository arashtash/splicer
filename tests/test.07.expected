Lorem ipsum dolor sit amet, consectetur 
adipiscing elit. Pellentesque sed 
dolor non arcu imperdiet placerat. 
Pellentesque ullamcorper urna vel 
magna pharetra, et placerat libero 
ultrices. Nunc ante ex, cursus 
non ultrices at, interdum vitae 
libero. Etiam nibh tortor, posuere 
ac tincidunt eget, commodo at nibh. 
Integer a fringilla lacus, tristique 
laoreet ligula. Pellentesque a 
sagittis neque. Curabitur molestie 
pharetra arcu, a rutrum libero 
tincidunt sed. Aliquam felis sapien, 
fringilla at varius nec, malesuada 
vel arcu. Fusce auctor mi et placerat 
dignissim. Fusce sit amet tellus 
mauris. Duis a lectus ligula. Nam 
viverra quam est, suscipit elementum 
ante porttitor iaculis. Nulla facilisi. 
Nam nec neque sed neque viverra 
congue id quis felis. Integer ultricies 
risus et viverra fermentum. Nam 
et metus dignissim, gravida lacus 
blandit, sodales est. Duis tincidunt, 
turpis eu posuere pellentesque, 
urna nisi accumsan ante, ut facilisis 
turpis felis in urna. In viverra 
sem eros, nec varius purus dapibus 
eu. Vestibulum eget nunc ac ligula 
dictum feugiat id at ligula. Class 
aptent taciti sociosqu ad litora 
torquent per conubia nostra, per 
inceptos himenaeos. 
